contracts/protocols/mainnet/compound/main.sol

```diff
@@ -6,7 +6,7 @@ import "./helpers.sol";
 contract Resolver is Helpers {
     function getPriceInEth(CTokenInterface cToken) public view returns (uint priceInETH, uint priceInUSD) {
         uint decimals = getCETHAddress() == address(cToken) ? 18 : TokenInterface(cToken.underlying()).decimals();
-        uint ethPrice = OrcaleComp(getOracleAddress()).price("ETH") * 1e12;
+        uint ethPrice = uint(ETH_PRICE_ORACLE.latestAnswer()) / 10 ** ETH_PRICE_ORACLE.decimals();
```

Comment:
@SamarendraGouda why are you converting the value to decimals? I guess it should be in 18 decimals